dbsync.py

```diff
@@ -284,13 +217,21 @@ def create_mergin_client():
         raise DbSyncError("Mergin Maps client error: " + str(e))
 
 
-def dbsync_pull(mc):
+def pull(conn_cfg, mc):
     """ Downloads any changes from Mergin Maps and applies them to the database """
 
-    _check_has_working_dir()
-    _check_has_sync_file()
+    print(f"Processing Mergin Maps project '{conn_cfg.mergin_project}'")
+    ignored_tables = ";".join(conn_cfg.skip_tables) if "skip_tables" in conn_cfg else ""
+
+    project_name = conn_cfg.mergin_project.split("/")[1]
+    work_dir = os.path.join(config.working_dir, project_name)
+    gpkg_full_path = os.path.join(work_dir, conn_cfg.sync_file)
+
+    _check_has_working_dir(work_dir)
+    _check_has_sync_file(gpkg_full_path)
 
-    mp = MerginProject(config.project_working_dir)
+    mp = MerginProject(work_dir)
+    mp.set_tables_to_skip(ignored_tables)
```

Comment:
I am wondering if there is a use case we need to remove/reset this settings or it is more like global settings for project in which case it might be better to pass it to constructor instead of having special method for that ...